resources/views/pages/profile/show.blade.php

```diff
@@ -120,6 +135,53 @@
     }).catch(function(error) {
         console.error(error);
     });
+
+    //hit media webservice and retrieve raw audio file
+    axios.get(
+        profileAudioUri,
+        {
+            baseURL: mediaWsUrl + 'faculty/media'
+        }
+    ).then(function(response) {
+        //check to make sure the audio file exists
+        if( response.data.audio_recording !== undefined && response.data.audio_recording !== null) {
+            //set the <audio> element src attribute to the audio file we just retrieved from media
+            $("#audio").attr('src',response.data.audio_recording + "?source=true")
+
+            //make the play icon visible and add event listener
+            audioPlayButton.classList.remove("d-none");
+            audioPlayButton.addEventListener("click", handlePlayButton, false);
+            
+            // the rest of this code uses the HTMLMediaElement API to handle playing, pausing, and ending the video
+            function handlePlayButton() {
+                if (audioElem.paused) {
+                    playAudio();
+                } else {
+                    audioElem.pause();
+                }
+            }
+            async function playAudio() {
+                try {
+                    await audioElem.play();
```

Comment:
Could this be simplified by tacking on .catch() after .play()?

`audioElem.play().catch(() => //something on error)`?

Reply:
no, unfortunately that produces a syntax error: "missing catch or finally after try"